docs/components/plugins/api/APISectionClasses.tsx

```diff
@@ -73,31 +74,33 @@ const renderClass = (clx: ClassDefinitionData, hasMultipleClasses: boolean): JSX
       {properties?.length ? (
         <>
           {hasMultipleClasses ? (
-            <>
+            <div css={STYLES_NESTED_SECTION_HEADER}>
               <H4>{name} Properties</H4>
-              <br />
-            </>
+            </div>
           ) : (
             <H2>{name} Properties</H2>
           )}
-          <UL>
+          <div>
             {properties.map(property =>
               renderProp(property, property?.defaultValue, !hasMultipleClasses)
             )}
-          </UL>
+          </div>
         </>
       ) : null}
-      {methods?.length ? (
+      {methods?.length && (
         <>
-          {hasMultipleClasses ? <H4>{name} Methods</H4> : <H2>{name} Methods</H2>}
-          <div style={{ paddingLeft: 8 }}>
-            {methods.map((method, index) =>
-              renderMethod(method, index, methods.length, undefined, undefined, !hasMultipleClasses)
-            )}
-          </div>
+          {hasMultipleClasses ? (
+            <div css={STYLES_NESTED_SECTION_HEADER}>
+              <H4>{name} Methods</H4>
+            </div>
+          ) : (
+            <H2>{name} Methods</H2>
+          )}
+          {methods.map((method, index) =>
+            renderMethod(method, index, methods.length, undefined, undefined, !hasMultipleClasses)
```

Comment:
This is outside the scope of this PR, but this method has two unused arguments (`index` and `dataLength`). It also has too many arguments and is not readable. Can we make another PR to update this to use an object?

```ts
export function renderMethod({ methodDefinition: ..., apiName: ..., header: ...,  ... }: RenderMethodOptions) {
  // ...
}
```

Reply:
Yeah, this method evolved in time, with the features added, and it is a good suggestion, will refactor this in separate PR. 👍 

Also just FYI, any of the `APISection` components or code is not considered final, until we finish the docs redesign, because we need to replace a lot of component used internally. Also I am still changing things with almost every new package converted. 😅